### PR TITLE
Deserialize empty order.order_class to Class::Simple.

### LINKED ISSUE #37: unknown variant ``, expected one of `simple`, `bracket`, `oco`, `oto`
I've seen the following failure a couple of times in CI now ([example](https://gitlab.com/d-e-s-o/apca/-/jobs/2400875526)):
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Endpoint(Conversion(Json(Error("unknown variant ``, expected one of `simple`, `bracket`, `oco`, `oto`", line: 1, column: 21996))))', src/api/v2/orders.rs:191:52
---- api::v2::orders::tests::list_orders stdout ----
```

This may be related to the order class change 569616cfed63d5b93b1e70a5dea8aae086cb7b73. I have not yet been able to reproduce locally, but haven't tried hard either. My suspicion is that leg orders don't have the order class property populated correctly, but I haven't verified that.

@husafan, have you seen this problem during your testing?

### PR DESCRIPTION
**Pull request by husafan**
Fixes #37.

### CODE REVIEW

**Comment by d-e-s-o** [comment]
This seems reasonable to me, thanks! But I think we probably should open an upstream issue for this problem. I honestly doubt an empty order class is intentional. Would you mind opening one?

File: src/api/v2/order.rs [hunk 1]
```diff
@@ -535,6 +535,17 @@ pub struct ChangeReq {
...
 
 
+fn empty_to_simple<'de, D>(deserializer: D) -> Result<Class, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let class = Class::deserialize(deserializer);
+    match class {
+      Ok(value) => Ok(value),
+      Err(_) => Ok(Class::Simple),
```
Comment (d-e-s-o) [inline src/api/v2/order.rs:545]: The one issue here is that we silently swallow all kinds of errors. We shouldn't be doing that, or we will mask other legitimate errors. But I already found a way to fix that.

**Comment by d-e-s-o** [comment]
I fix this one problem and merged the change. Thanks again for looking into these failures!

**Comment by husafan** [comment]
I've opened an issue here: https://github.com/alpacahq/Alpaca-API/issues/232

**Pull request closed without merging**